scene/gui/color_picker.cpp

```diff
@@ -535,7 +537,15 @@ void ColorPicker::_preset_input(const Ref<InputEvent> &p_event) {
 	if (bev.is_valid()) {
 
 		if (bev->is_pressed() && bev->get_button_index() == BUTTON_LEFT) {
-			int index = bev->get_position().x / (preset->get_size().x / presets.size());
+			
```

Comment:
There's trailing whitespace here, and in other places you have mixed use of tabs and spaces (it should all be tabs for indentation). See the `clang-format` diff on https://travis-ci.org/godotengine/godot/jobs/555661957 (select to see whitespace).

Reply:
See also http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html